zebra/zebra_vxlan.c

```diff
@@ -4038,6 +4038,15 @@ int zebra_vxlan_dp_network_mac_add(struct interface *ifp,
 	struct zebra_evpn_es *es;
 	struct interface *acc_ifp;
 
+	/* If netlink message with vid for vxlan, it has no vxlan's nexthop.
+	 * Skip it.
+	 */
+	if (vid)
```

Comment:
Add a debug here so we know why we didn't do anything with the message.

Reply:
Done.